ui/app/components/ui/list-item/index.scss

```diff
@@ -8,32 +8,28 @@
   border-top: 1px solid $mercury;
   border-bottom: 1px solid $mercury;
   color: $Black-100;
-
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
+  display: grid;
+  grid-template-columns: 0fr repeat(11, 1fr);
+  grid-template-areas:
+    'icon head     head     head     head     head     head     head     right right right right'
```

Comment:
my first draft was using letters 'i, h, s, a, r, m' instead of words. Let me know what you prefer.

Reply:
The words are definitely much better!